dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImpl.java

```diff
@@ -1631,7 +1629,40 @@ SearchHits cachedIndexSearch(final SearchRequest searchRequest) {
         
         
     }
-    
+
+    /**
+     * if enabled CountRequest are executed and then cached
+     * @param countRequest
+     * @return
+     */
+    Long cachedIndexCount(final CountRequest countRequest) {
```

Comment:
Do we need this if we are getting the correct total back from the SearchHits?

Reply:
This one is always accurate. and does not depend on the `track_total_hits`